cmd/sysbox-fs/main.go

```diff
@@ -187,7 +187,7 @@ func main() {
 		},
 		cli.StringFlag{
 			Name:  "log",
-			Value: "/dev/stdout",
+			Value: "/var/log/sysbox-fs.log",
 			Usage: "log file path",
```

Comment:
The usage hint for logging should probably include the hint that using "" will redirect to os.Stderr

Reply:
You are right. But to be true, this output is for people starting sysbox-fs and sysbox-mgr manually from console. I assume those people expect Stderr to be the default and not /var/log/sysbox-fs/mgr.log. So the default should be "". But this looks strange when people run -h or --help because the Value is printed out there and also used as default value. And before my fix the Value was the non working /dev/stdout so i think my patch was not so backward compatible then. But i think we can change it to "" while adding documentation. And the systemd sets this option and is independent of the default value. I try.